<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mtplayClass</class>
 <widget class="QWidget" name="mtplayClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mtplay</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(161, 187, 215);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item row="0" column="0" colspan="6">
    <widget class="Mt_VideoWidget" name="video">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="btn_open">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/mtplay/im/open.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="btn_play">
     <property name="styleSheet">
      <string notr="true">border-image: url(:/mtplay/im/play.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QSlider" name="mySlider">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="maximum">
      <number>999</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label_time">
     <property name="text">
      <string>000:00</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="label_shu">
     <property name="text">
      <string>|</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QLabel" name="label_timeplay">
     <property name="text">
      <string>000:00</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Mt_VideoWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>Mt_VideoWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="mtplay.qrc"/>
 </resources>
 <connections/>
</ui>
